packages/terra-demographics-banner/src/DemographicsBanner.module.scss

```diff
@@ -115,4 +136,64 @@
       text-align: right;
     }
   }
+
+  .is-selectable {
+    cursor: pointer;
+    display: flex;
+    flex-direction: row;
+
+    &:focus {
+      outline: none;
+    }
+    
+    &:focus[data-focus-styles-enabled=true] {
+      box-shadow: var(--terra-demographics-banner-focus-box-shadow, none);
+      outline: var(--terra-demographics-banner-focus-outline, 2px dashed #fff);
+      outline-offset: var(--terra-demographics-banner-focus-outline-offset, -5px);
+    }
+
+    &::-moz-focus-inner {
+      border: 0;
+      padding: 0;
+    }
+
+    &:hover {
```

Comment:
We may need to scope these hover styles to mouse/pointer only (based on https://github.com/cerner/terra-core/pull/2932)